packages/backend/src/test.rs

```diff
@@ -0,0 +1,21 @@
+use sqlx::PgPool; // Import PgPool for PostgreSQL connection
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<(), sqlx::Error> {
+    // Set up the database connection pool
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+    let pool = PgPool::connect(&database_url).await?;
+
+    // Execute the query and fetch results
+    let rows = sqlx::query("SELECT * FROM Catcolab LIMIT 5")
+        .fetch_all(&pool)
+        .await?;
+
+    // Print each row
+    for row in rows {
+        println!("{:?}", row);
+    }
+
+    Ok(())
+}
```

Comment:
Where does the 'Catcolab' table come from? (also why is there a capital in a table name, and if there is, it needs to be wrapped in quotes)

This test will always pass, which isn't much of a test.